src/vuestic-theme/vuestic-components/vuestic-simple-select/VuesticSimpleSelect.vue

```diff
@@ -9,9 +9,16 @@
         :class="{'has-value': !!value}"
         v-model="displayValue"
         :name="name"
-        :options="options"/>
-      <i class="ion ion-ios-arrow-down icon-right input-icon"></i>
-      <label class="control-label">{{label}}</label><i class="bar"></i>
+        :options="options"
+        :placeholder="selectedValue">
+      <div v-if="clearable && selectedValue !== '' && displayValue !== '' && selectedValue !== undefined">
```

Comment:
Hard to read / understand. Move this into computed property maybe:
`v-if="clearable && hasSelected && hasDisplay"`